packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -178,13 +178,14 @@ class BySemanticsLabel extends SerializableFinder {
   }
 }
 
-/// A Flutter Driver finder that finds widgets by [text] inside a [Text] or
-/// [EditableText] widget.
+/// A Flutter Driver finder that finds widgets by [text] inside a
+/// [widgets.Text] or [widgets.EditableText] widget.
```

Comment:
Will `[widgets.Text]` print `widgets.Text`, or just `Text`?

Reply:
It will display the library, but I think dart-lang/dartdoc#2713 is a good idea to stop doing that.

More broadly, we could consider having some sort of library or package specific specifier indicating how to resolve ambiguous links.  I'm less sure what the right solution there is, though.